canonical-webteam/documentation-builder
# Add app + Dockerfile to serve documentation

Add Python app to serve documentation from the docs folder and a Dockerfile to build deployment image

## QA

- `cd docs`
- `./run` - Visit site, check output

Then build and run the Docker image
- `docker build --build-arg COMMIT_ID=test --tag builder-docs .`
- ` docker run --rm -p 80:80 -it builder-docs `
